spec/features/admin/dashboard_spec.rb

```diff
@@ -2,13 +2,14 @@
 
 RSpec.feature "Admin Dashboard" do 
   it "has a header" do 
-    visit '/admin'
+    visit admin_path
+    # visit '/admin'
 
```

Comment:
we can probably get rid of this line since you have the resource path up and working!:)